src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -1128,10 +1141,6 @@ private void duplicateCurrentlySelectedModElement() {
 						duplicateModElement.setCodeLock(true);
 					}
 
-					// if we are not in the root folder, specify the folder of the mod element
-					if (!currentFolder.equals(mcreator.getWorkspace().getFoldersRoot()))

```

Comment:
If there is no good reason to change location, I would keep this here to make PR more readable.

I like this position more as it is just before addModElement call

Reply:
So move the new call here?

Reply:
:heavy_check_mark: Fixed